Header header
uint8 vehicleID
uint8 type
uint8 packetNumber
uint8 seqNum
float64[3] accelerometer
float64[3] rateGyro
float64[3] position
float64[3] attitude
float64[3] velocity
float64[3] attitudeYPR
float64[4] motorForces
float64[6] debugVals
float64 batteryVoltage
uint8 panicReason
uint8 warnings
float32[14] customPacket1
float32[14] customPacket2
# PX4-ESP specific
float32 voltage
float32 current
uint8 temperature
uint16[4] esc_rpm
bool[8] status_bits
float32[8] esp_debug_vals